tests_e2e/tests/scripts/agent_persist_firewall-access_wireserver

```diff
@@ -48,6 +66,19 @@ while [ "$IS_ONLINE" -eq 1 ]; do
 
 done
 
+# logging other options output to compare and evaluate which option is more stable when ping to 8.8.8.8 failed
+if [ "$IS_ONLINE" -eq 1 ]; then
```

Comment:
Just confirming we're only logging these options right now. It doesn't look like we update IS_ONLINE with the results of these options

Reply:
Just logging for now, we look at these outputs when test fail with first option.